msal/src/main/java/com/microsoft/identity/client/internal/controllers/MSALTokenCommand.java

```diff
@@ -60,9 +64,51 @@ public MSALTokenCommand(@NonNull final Context context,
         }
     }
 
+    public MSALTokenCommand(@NonNull final Context context,
+                            @NonNull final MSALOperationParameters parameters,
+                            @NonNull final List<MSALController> controllers,
+                            @NonNull final AuthenticationCallback callback) {
+        mContext = context;
+        mParameters = parameters;
+        mController = null;
+        mControllers = controllers;
+        mCallback = callback;
+
+        if (!(mParameters instanceof MSALAcquireTokenSilentOperationParameters)) {
```

Comment:
Is this check needed? Or could we change the constructor args to explicitly require this type?

```java
public MSALTokenCommand(@NonNull final Context context,
                            @NonNull final MSALAcquireTokenSilentOperationParameters parameters,
                            @NonNull final List<MSALController> controllers,
                            @NonNull final AuthenticationCallback callback) {
    // ...
}
```

Reply:
This part isn't actually new in this PR.